main.tf

```diff
@@ -1,4 +1,5 @@
-// Copyright (c) 2018, 2021, Oracle and/or its affiliates.
+# Copyright (c) 2018, 2021 Oracle Corporation and/or affiliates.  All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl
 
 terraform {
   required_version = ">= 0.12" // terraform version below 0.12 is not tested/supported with this module
```

Comment:
Isn't 0.12 no longer supported?

Reply:
Terraform 0.11 is deprecated, but 0.12 is not yet. Moving to Terraform 1.0 is part of module v3.0.0.
For now, I just want to get v2.2.0 out as there is a decent list of feature that are hold back by this.